Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Dockerfile
+# A blueprint for building a docker image
+FROM python:3.9
+
+# ADD move_embargo_scratch.py .
+
+# RUN setup lsst_distrib -t w_2023_19
+
+FROM lsstsqre/newinstall:latest
+USER lsst
+RUN source loadLSST.bash && mamba install redis-py rucio-clients
+RUN source loadLSST.bash && eups distrib install -t "w_2023_21" obs_lsst
+
+# RUN pip install -r requirements.txt
+
```

Comment:
You'll need something like `COPY ./src /opt/lsst/transfer_embargo` and maybe a `WORKDIR /opt/lsst/transfer_embargo` as well.  See https://docs.docker.com/engine/reference/builder/ for everything you can use.

Reply:
We changed these, but have some questions about this path.